sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb

```diff
@@ -51,7 +51,7 @@
    "metadata": {},
```

Comment:
<p>Profiling is now available in the default TF 2.3.1 images. Can remove <code>image_uri</code> and uncomment <code>framework_version</code> / <code>py_version</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1954/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

Reply:
incorporated